src/lang/de.rs

```diff
@@ -564,5 +564,6 @@ pub static ref T: std::collections::HashMap<&'static str, &'static str> =
         ("elevated_switch_display_msg", "Wechseln Sie zur primären Anzeige, da die Mehrfachanzeige im erweiterten Modus nicht unterstützt wird."),
         ("Open in new window", "In einem neuen Fenster öffnen"),
         ("Show displays as individual windows", "Anzeigen als einzelne Fenster darstellen"),
+        ("Use all my displays for the remote session", "Alle meine Anzeigen für die Fernsitzung verwenden"),
```

Comment:
wouldn't "Alle meine Monitore..." be better here? This sounds more like a Google translation.

Reply:
Just thoughts. 

For constistency, some other "display"/"Anzeige" should changed to "Monitor" or "Bildschirm" then.

"Anzeigen" is more common as a verb, like:

```
("show_monitors_tip", "Monitore in der Symbolleiste anzeigen"),
```

```
("Show displays as individual windows", "Anzeigen als einzelne Fenster darstellen"),
```

- Monitore in eigenem Fenster anzeigen.
- Jeden Monitor in eigenem Fenster anzeigen.
- Alle Bildschirme in eigenem Fenster anzeigen.
- Ein eigenes Fenster für jeden Monitor [öffnen].
- Für jeden Bildschirm ein eigenes Fenster öffnen.

```
("elevated_switch_display_msg", "Wechseln Sie zur primären Anzeige, da die Mehrfachanzeige im erweiterten Modus nicht unterstützt wird."),
```

- Wechseln Sie zum primären Bildschirm, …

```
("switch_display_elevated_connections_tip", "Das Umschalten auf eine nicht primäre Anzeige wird mit erhöhten Rechten nicht unterstützt, wenn mehrere Verbindungen bestehen. Bitte versuchen Sie es nach der Installation erneut, wenn Sie mehrere Anzeigen steuern möchten."),
```

- […] Umschalten auf einen sekundären/weiteren Monitor wird […] mehrere Bildschirme steuern […] 

```
("Display", "Anzeige"),
```

- Monitor / Bildschirm

Reply:
I hope I have found an optimal translation.

Reply:
I think you did. Thanks.